src/rpc/server.h

```diff
@@ -46,8 +48,15 @@ class JSONRPCRequest
     bool fHelp;
     std::string URI;
     std::string authUser;
-
-    JSONRPCRequest() : id(NullUniValue), params(NullUniValue), fHelp(false) {}
+#ifdef ENABLE_WALLET
```

Comment:
In commit "RPC: Pass wallet through JSONRPCRequest"

Consider dropping this ifdef. It saves an insignificant amount of memory and seems like noise.